.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -60,6 +60,13 @@ sqs_CreateQueue:
             - description: Create an &SQS; queue that waits for a message to arrive.
               snippet_tags:
                 - sqs.JavaScript.longPoll.createQueue
+    Go:
+      versions:
+        - sdk_version: 2
+          github: gov2/sqs
+          excerpts:
+            - description: Create an &SQS; queue.

```

Comment:
It's your call, but in general the `description` field is best used to add info that's specific to the Go example. In this case, you're only restating the title/synopsis. It's a bit redundant, so I'd remove them.